input/fsh/QuestionnaireSEVendorLite.fsh

```diff
@@ -0,0 +1,84 @@
+Profile: QuestionnaireSEVendorLite
+Parent: Questionnaire
+Description: "Questionnaire with restrictions for swedish vendors."
```

Comment:
"Abstract Questionnaire with restrictions for swedish vendors." ??

Reply:
Rewrote the description a bit